src/project/qml/MuseScore/Project/internal/ChooseTemplatePage.qml

```diff
@@ -88,6 +90,11 @@ Item {
                 model.setCurrentTemplate(index)
             }
 
+            onDoubleClicked: {
```

Comment:
Please capture the `index` parameter explicitly here: 
```suggestion
            onDoubleClicked: function(index) {
```
Currently, it is not done consistently in the code, but we really should be doing it at least in new code. Why? To save me some work when making MuseScore ready for Qt 6, where capturing parameters is mandatory. See #10108; 16 of the 75 commits there are about this subject.

Reply:
Alright, will keep this in mind moving forward